README.md

```diff
@@ -18,6 +18,47 @@ This project is a template for [React Native](https://reactnative.dev/) that can
 
 NodeJS (version 16 or later) and NPM is required.
 To make sure you have them available on your machine, try running the following command
+## About Us
+
+Welcome to Awesome App! We're a small team of developers passionate about creating library that make people's lives easier.
+
+### Team Members
+
+Our team consists of 14 developers with diverse backgrounds:
+
+- Hong Hoang: Our lead developer with over 10 years of experience in software development and 5 years of experience in mobile application (react native). He's responsible for the overall architecture and technical implementation of the app.
+- Chung Nguyen: 
+- Phi Tran: 
+- Vu Nguyen: 
+- Huy Duong: 
+- Anh Le Hoang: 
+- Thinh Vo: 
+- Loc Nguyen: 
```

Comment:
with 5 years working in Front-end Development and over 4 years experience with Mobile Development (React Native).